### PR TITLE
Tabs - New component

### LINKED ISSUE #18: [Component] - Tabs

### PR DESCRIPTION
**Pull request by alemesa**
Resolve #18

### CODE REVIEW

File: src/components/Tabs/Tabs.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  function handleKeyup(e) {
+    e.preventDefault();
+    if (e.which === keys.left) previousTab();
+    else if (e.which === keys.right) nextTab();
+    else if (e.which === keys.end) setActive(childrenArray.length - 1);
+    else if (e.which === keys.home) setActive(0);
+  }
+
+  return (
+    <div className="Tabs" ref={containerRef}>
```
Comment (realdeemy) [inline src/components/Tabs/Tabs.js:38]: use `nav` tag

File: src/components/Tabs/Tabs.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      <ol className="tabs-list" role="tablist">
```
Comment (realdeemy) [inline src/components/Tabs/Tabs.js]: any reason for using `ol` vs `ul` here?

File: src/components/Tabs/Tabs-story.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    () => (
+      <Tabs>
+        <p label="First" style={{ ...commonStyles }}>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime omnis consequuntur, obcaecati reiciendis porro
+          doloremque natus dolores eum consequatur delectus aspernatur est dignissimos cupiditate ex sequi laudantium
+          facere qui? Maxime. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt officia odit possimus
+          inventore quas doloremque quaerat porro quis, enim magnam natus tempore numquam quia veritatis, quae hic odio
+          molestias iure.
+        </p>
+      </Tabs>
```
Comment (realdeemy) [inline src/components/Tabs/Tabs-story.js]: `label` attribute isn't valid

File: src/components/Tabs/Tabs.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  useEffect(() => {
+    if (isActive) el.current.focus();
+  }, [isActive]);
+
+  return (
+    <li
+      ref={el}
+      className={classnames('Tab', { active: isActive })}
+      role="tab"
+      aria-controls={label}
```
Comment (realdeemy) [inline src/components/Tabs/Tabs.js]: looks like `aria-controls` attribute is invalid

according to this, it's supposed to be an element id
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role

image

**Review by realdeemy — CHANGES_REQUESTED** [review]
@alemesa thanks fo taking care of it.

left a few comments

File: src/components/Tabs/Tabs.js (hunk 4 above)
Comment (alemesa) [reply src/components/Tabs/Tabs.js]: awesome, I fixed this

**Comment by alemesa** [comment]
Screen Shot 2020-09-03 at 12 36 57 PM

Thanks @namniak I resolved some comments and added more a11y improvements

**Review by jmckinnell — APPROVED** [review]
Looks good from my end. @namniak Review when you can.

**Comment by realdeemy** [comment]
@alemesa amazing, man! 💯

**Review by realdeemy — APPROVED** [review]

**Pull request merged**